x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.ts

```diff
@@ -56,6 +64,23 @@ export const getPackagePolicyCreateCallback = (
       return newPackagePolicy;
     }
 
+    // Optional endpoint integration configuration
+    let endpointIntegrationConfig;
+
+    // Check if has endpoint integration configuration input
+    const integrationConfigInput = newPackagePolicy?.inputs?.find(
+      (input) => input.type === ENDPOINT_INTEGRATION_CONFIG_KEY
+    )?.config?._config;
+
+    if (integrationConfigInput !== undefined) {
```

Comment:
suggestion: change to `!integrationConfigInput` - just in case some other falsy value is used.